# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/jetporch/hotfix-version.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- jetporch-40f8151d507ad136cd0eb7819703dbe548889bb1/version.sh.vanilla	2026-01-05 21:33:24.888673392 +0100
+++ jetporch-40f8151d507ad136cd0eb7819703dbe548889bb1/version.sh	2026-01-05 21:33:46.451966280 +0100
@@ -1,11 +1,9 @@
 #!/bin/sh
 set -e
 
-head=`git rev-parse HEAD`
-branch=`git rev-parse --abbrev-ref HEAD` 
 date=`date`
 
 echo "// auto generated by version.sh script" > src/cli/version.rs
-echo "pub const GIT_VERSION: &str  = \"$head\";" >> src/cli/version.rs
-echo "pub const GIT_BRANCH: &str  = \"$branch\";" >> src/cli/version.rs
+echo "pub const GIT_VERSION: &str  = \"$ver\";" >> src/cli/version.rs
+echo "pub const GIT_BRANCH: &str  = \"$ver\";" >> src/cli/version.rs
 echo "pub const BUILD_TIME: &str  = \"$date\";" >> src/cli/version.rs
